### PR TITLE
Implement small UI tweaks

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

1. Make drawers feel nicer (snappier when they are shorter)
2. Use touchable opacity on track tiles

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/mobile/src/components/drawer/Drawer.tsx [hunk 1]
```diff
@@ -249,8 +251,11 @@ export const springToValue = ({
       friction = 25
       break
     case DrawerAnimationStyle.SPRINGY:
-      tension = 70
-      friction = 10
+      // Factor the height of the drawer into the spring physics.
+      // Without this, short drawers tend to feel sluggish while
+      // tall drawers really get going.
+      tension = 70 + 60 * (1 - drawerHeight / FULL_DRAWER_HEIGHT)
```
Comment (dylanjeffers) [inline packages/mobile/src/components/drawer/Drawer.tsx:257]: Love this change!

File: packages/mobile/src/components/drawer/Drawer.tsx [hunk 2]
```diff
@@ -249,8 +251,11 @@ export const springToValue = ({
       friction = 25
       break
     case DrawerAnimationStyle.SPRINGY:
-      tension = 70
-      friction = 10
+      // Factor the height of the drawer into the spring physics.
+      // Without this, short drawers tend to feel sluggish while
+      // tall drawers really get going.
```
Comment (nicoback2) [inline packages/mobile/src/components/drawer/Drawer.tsx:256]: 🔥

File: packages/mobile/src/components/lineup-tile/LineupTileStats.tsx [hunk 3]
```diff
@@ -134,8 +134,8 @@ export const LineupTileStats = ({
               fill={neutralLight4}
               style={[styles.statIcon, styles.repostStat]}
             />
-          </Pressable>
-          <Pressable
```
Comment (dylanjeffers) [inline packages/mobile/src/components/lineup-tile/LineupTileStats.tsx:138]: Word

**Review by dylanjeffers — APPROVED** [review]
Awesome changes dood

**Review by nicoback2 — APPROVED** [review]

**Pull request merged**